buildbot/configure.py

```diff
@@ -198,6 +201,13 @@ def do_configure(args):
             "-DSYCL_LIBCXX_INCLUDE_PATH={}".format(args.libcxx_include),
             "-DSYCL_LIBCXX_LIBRARY_PATH={}".format(args.libcxx_library)])
 
+    if args.default_openmp:
+        omp_dir = os.path.join(abs_src_dir, "openmp")
+        cmake_cmd.extend([
+            "-DLLVM_EXTERNAL_OPENMP_SOURCE_DIR={}".format(omp_dir),
+            "-DOPENMP_ENABLE_LIBOMPTARGET=OFF",
+            "-DLLVM_ENABLE_PIC=ON"])
```

Comment:
Why do we need those two? Also, per https://llvm.org/docs/CMake.html#options-and-variables,

> LLVM_ENABLE_PIC:BOOL
Add the -fPIC flag to the compiler command-line, if the compiler supports this flag. Some systems, like Windows, do not need this flag. Defaults to ON.

Reply:
Hi @aelovikov-intel, 

You're right. It is not strictly necessary. I guess there were some cases where I had to compile with -fPIC
but I guess it was not  related to OpenMP.
Can I ask you whether I can make `DLLVM_ENABLE_PIC=ON` optional? For example, I can add an option
`--enable-pic`

Reply:
Well, I don't see why you'd need that. Can I ask you for a reproducer where something doesn't work if you don't add it to the cmake configuration command?

Reply:
Hi @aelovikov-intel, 

I just tested again and, yes, `DLLVM_ENABLE_PIC=ON` is not necessary. I will remove it